setup.cfg

```diff
@@ -0,0 +1,12 @@
+[flake8]
```

Comment:
Given we use setup.py instead of setup.cfg for package installation, perhaps it's clearer if flake8 is configured in `.flake8` rather than `setup.cfg` or `tox.ini`? (https://flake8.pycqa.org/en/latest/user/configuration.html)